components/vr-image.js

```diff
@@ -0,0 +1,51 @@
+/* globals define */
+(function(define){'use strict';define(function(require,exports,module){
+
+  document.registerElement(
+    'vr-image',
+    {
+      prototype: Object.create(
+        VRObject.prototype, {
+          init: {
+            value: function() {
+              var material = this.getMaterial();
+              var geometry = this.getGeometry();
+              this.object3D = new THREE.Mesh( geometry, material );
+              this.load();
+            }
+          },
+
+          update: {
+            value: function() {
+              var material = this.getMaterial();
+              var geometry = this.getGeometry();
+              this.object3D = new THREE.Mesh( geometry, material );
```

Comment:
You don't have to recreate the object each time that the attributes change. You should do: 
```javascript
  this.object3D.geometry = geometry;
  this.object3D.material = material;
```